test/AutoRest.TestServerLowLevel.Tests/RequestContextTests.cs

```diff
@@ -21,14 +21,14 @@ public class RequestContextTests : TestServerLowLevelTestBase
         [Test]
         public Task RequestThrowsByDefault () => Test(host =>
         {
-            PathsClient paths = new PathsClient(Key, host, null);
+            Paths paths = new AutoRestUrlTestServiceClient(Key, host, null).GetPathsClient();
```

Comment:
so if `single-top-level-client : true`, there will always be a top level client?

Reply:
yes

Reply:
@chunyu3, why is `PathsClient` not the top level client?

Reply:
In Url RP in the test-server, there are more than one operation groups (paths, queries,pathItems). So when set 'single-top-level-client', it will create a new service client and use the title as the client name, all the operation groups will be subclient.